### PR TITLE
Rewrite introduction

### LINKED ISSUE #186: Review introduction to spec
also consider where we should put information on prior art and history of this spec.

### PR DESCRIPTION
**Pull request by rosy1280**
fixes #186

### CODE REVIEW

File: draft/spec/index.html [hunk 1]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
         <p>
-          A general observation is that the contents of a digital repository &#8212; that is, the digital files and
-          metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
-          accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
-          applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
-          Thus, transitions between application-specific methods of file management to support software upgrades and
-          replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
-          in support of the short-term, ephemeral software.
+          The OCFL recommends storing metadata and the content it describes together so the OCFL object can be fully
+          understood in the absence of original software. OCFL does not make recommendations about what constitutes an
```
Comment (zimeon) [inline draft/spec/index.html]: **The** OCFL does not...

File: draft/spec/index.html [hunk 2]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          object, nor does it assume what type of metadata is needed to fully understand the object recognizing those
```
Comment (zimeon) [inline draft/spec/index.html]: add comma before "recognizing"? I needed a breath

File: draft/spec/index.html [hunk 3]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
         </p>
+        <h3>Parsability</h3>
         <p>
-          By providing a specification for the file and directory layout on disk or in an object store, the OCFL is an
-          attempt at reducing, or even eliminating, the need for these transitions. As an application-independent
-          specification, conforming applications will natively "understand" the underlying file structure without
-          needing to first transition these contents to their own format.
-         </p>
+          One goal of the OCFL is to ensure objects remain fixed over time. This can be difficult as software and
+          infrastructure change and content is migrated. To combat this challenge, the OCFL ensures that both humans
```
Comment (zimeon) [inline draft/spec/index.html]: comma before "and content" to separate list from from previous "and"

File: draft/spec/index.html [hunk 4]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          and machines can understand the layout and corresponding inventory regardless of the software or
+          infrastructure used. This allows for humans to read the layout and corresponding inventory and understand it
```
Comment (zimeon) [inline draft/spec/index.html]: again, comma before "and understand"

File: draft/spec/index.html [hunk 5]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          without the use of machines. Additionally, if software were to become obsolete, the OCFL could easily be
```
Comment (zimeon) [inline draft/spec/index.html]: if **existing** software were to become obsolete?

File: draft/spec/index.html [hunk 6]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          in the past.
+        </p>
+        <h3>Versioning</h3>
         <p>
-          While digital repository content changes relatively slowly, it is necessary to be able to track changes to
-          digital objects. Within the file and directory layout specification, the OCFL provides a simple structure to
-          efficiently capture versions of object contents so that all previous states of an object may be recovered
-          and examined.
+          Another need expressed by the community was the need to update and change objects, either the content itself
+          of the metadata associated with the object. The OCFL relies heavily on the prior art in the [[Moab]] Design
```
Comment (zimeon) [inline draft/spec/index.html]: of -> or

File: draft/spec/index.html [hunk 7]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          this schema allows implementers of the OCFL to easily recreate past versions of an OCFL object. Like with
+          determining what constitutes an OCFL object, the OCFL remains silent on when versioning should occur
+          recognizing again that this may differe from implementation to implementation.
+        </p>
+        <h3>Robustness</h3>
+        <p>
+          The OCFL also fills the need for robustness against errors, corruption, and migration. The versioning schema
+          ensures an OCFL object is robust enough to allow for the discovery of human errors. The fixity checking built
+          into OCFL via content addressable storage allows implementers to identify file corruption that might happen
+          outside of normal human interactions. Finally, the OCFL allows for methods that ease complications related to
```
Comment (zimeon) [inline draft/spec/index.html]: Can the "Finally.." sentence be made more specific? I'm not sure what this means

Comment (zimeon) [reply draft/spec/index.html]: also, should not use "Finally" because the next section starts with that

Comment (rosy1280) [reply draft/spec/index.html]: @zimeon check to see if this change is better

Comment (zimeon) [reply draft/spec/index.html]: Yes except "The OCFL ~allows~ eases..."

File: draft/spec/index.html [hunk 8]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          for Digital Object Versioning which utilizes forward deltas to track the history of the object. Utilizing
+          this schema allows implementers of the OCFL to easily recreate past versions of an OCFL object. Like with
+          determining what constitutes an OCFL object, the OCFL remains silent on when versioning should occur
+          recognizing again that this may differe from implementation to implementation.
+        </p>
+        <h3>Robustness</h3>
+        <p>
+          The OCFL also fills the need for robustness against errors, corruption, and migration. The versioning schema
+          ensures an OCFL object is robust enough to allow for the discovery of human errors. The fixity checking built
+          into OCFL via content addressable storage allows implementers to identify file corruption that might happen
```
Comment (awoods) [inline draft/spec/index.html]: "OCFL" -> "the OCFL"

File: draft/spec/index.html [hunk 9]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
+          outside of normal human interactions. The OCFL eases content migrations by providing a technology agnostic
+          method for verifying OCFL objects have remained fixed.
+        </p>
+        <h3>Storage diversity</h3>
+        <p>
+          Finally, the community expressed a need to store content on a wide variety of storage technologies currently
```
Comment (awoods) [inline draft/spec/index.html]: Suggestion: remove the phrase: "currently available on the market".

File: draft/spec/index.html [hunk 10]
```diff
@@ -172,47 +172,74 @@ <h2>Introduction</h2>
...
-          While digital repository content changes relatively slowly, it is necessary to be able to track changes to
-          digital objects. Within the file and directory layout specification, the OCFL provides a simple structure to
-          efficiently capture versions of object contents so that all previous states of an object may be recovered
-          and examined.
+          Another need expressed by the community was the need to update and change objects, either the content itself
+          or the metadata associated with the object. The OCFL relies heavily on the prior art in the [[Moab]] Design
+          for Digital Object Versioning which utilizes forward deltas to track the history of the object. Utilizing
+          this schema allows implementers of the OCFL to easily recreate past versions of an OCFL object. Like with
+          determining what constitutes an OCFL object, the OCFL remains silent on when versioning should occur
+          recognizing again that this may differe from implementation to implementation.
```
[the remark this answers is not in the document]
Comment (rosy1280) [reply draft/spec/index.html]: Please review changes @awoods

**Pull request merged**